src/Services/Air/Air.js

```diff
@@ -15,6 +15,20 @@ module.exports = (settings) => {
       return service.searchLowFares(options);
     },
 
+    fareRules(options) {
```

Comment:
> **contains information about applying the farebasis to all PTCs and are linked to the fare basis itself**

That means that if you have fare basis which is available for `CNN` PTC only, then you will not be available to fetch them using `ADT` PTC in request.